operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -496,7 +496,7 @@ alertmanager:
     # If defined, storageClassName: <storageClass>
     # If set to "-", storageClassName: "", which disables dynamic provisioning
     # If undefined (the default) or set to null, no storageClassName spec is
-    #   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    #   set, choosing the default provisioner.
     #   GKE, AWS & OpenStack)
```

Comment:
This became a stray line, also in the other 3

Reply:
Oh :( Fixing.

Reply:
Apologies, fixed. PTAL.